lumispy/utils/signals.py

```diff
@@ -79,3 +98,172 @@ def _interpolate_signal(axis_array, index, **kwargs):
         raise ValueError("The parmeter `signal_axis` must be a HyperSpy Axis object.")
 
     return com_val
+
+
+#
+# navigation axis manipulation
+#
+
+
+def crop_edges(
+    S,
+    crop_range=None,
+    rebin_nav=False,
+    **kwargs,
+):
+    """
+    Cropping along the navigation axes of a list of signal objects.
+    Crop the amount of pixels from the four edges of the scanning
+    region, from the edges inwards. Cropping can happen uniformly on all
+    sides or by specifying the cropping range for each axis or each side. If the navigation axes shape is different, all signals can be rebinned to match the shape of the first signal in the list.
```

Comment:
Continue with line breaks for readability across the whole docstring and also in error messages below.